brainscore_vision/models/bp_resnet50_julios/model.py

```diff
@@ -23,16 +23,15 @@ def get_model_list():
 
 def get_model(name):
     assert name == 'bp_resnet50_julios'
-    model = torch.load('brainscore_vision/models/bp_resnet50_julios/bp_export.pt')
-    # download_weights(
-    # bucket='brainscore-vision', 
-    # folder_path='models/resnet50-deepdive-2',
-    # filename_version_sha=[('ckpt.pth', 'C722T4BityNPpazdXWiAeu8pGBxKIudb', '0186929df5d04451995d94cd332a3603a00594fe')],
-    # save_directory=Path(__file__).parent)
-    # ckpt_path = os.path.join(os.path.dirname(__file__), 'ckpt.pth')
-    # state_dict = torch.load(ckpt_path, map_location=torch.device('cpu'))
-    # model.load_state_dict(state_dict)
-    
+    ckpt_name = 'bp_export.pt'
+    download_weights(
+        bucket='brainscore-vision',
+        folder_path='models/bp_resnet50_julios',
+        filename_version_sha=[(ckpt_name, 'PAH2BMGhcw9tWUH6RwTIADHodw.fyG4G', 'e504fc533aaec453f24e73a86e47ca6df3394c53')],
+        save_directory=Path(__file__).parent
+    )
+    ckpt_path = os.path.join(os.path.dirname(__file__), ckpt_name)
```

Comment:
```suggestion
    ckpt_path = Path(__file__).parent / ckpt_name
```
just a suggestion if you find this to be nicer